apis/metal3.io/v1alpha1/baremetalhost_validation.go

```diff
@@ -53,3 +60,24 @@ func validateRAID(r *RAIDConfig) error {
 
 	return nil
 }
+
+func validateDNSName(hostaddress string) error {
+
+	if hostaddress == "" {
```

Comment:
Is this valid?

Reply:
This is to validate empty BareMetalHostSpec, but there could be a more precise check - maybe check if address is specified when Spec is not empty ?

Reply:
I mean; if hostaddress is empty, should we return error or return nil

Reply:
I can potentially see some value in creating a Host with no address and then specifying the address later. I can't think of any particular use cases though... the BMO can't do anything until there is an address.
tl;dr I don't know :laughing:

Reply:
It might be safe to return nil then :)